Client.UnitTests/JobWorkerTest.cs

```diff
@@ -274,7 +274,7 @@ public void ShouldSendRequestsWithDifferentAmounts()
             var expectedSecondRequest = new ActivateJobsRequest
             {
                 Timeout = 123_000L,
-                MaxJobsToActivate = 2,
+                MaxJobsToActivate = 1,
```

Comment:
I assume this was necessary because you removed the threshold.

Reply:
Yes